packages/camera/test/camera_test.dart

```diff
@@ -0,0 +1,49 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+import 'package:camera/camera.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('camera', () {
+    test('debugCheckIsDisposed should not throw assertion error when disposed',
+        () {
+      final MockCameraDescription description = MockCameraDescription();
+      final CameraController controller = CameraController(
+        description,
+        ResolutionPreset.low,
+      );
+
+      controller.dispose();
+
+      try {
```

Comment:
I believe [returnsNormally](https://pub.dev/documentation/matcher/latest/matcher/returnsNormally-constant.html) is the standard way to make sure no Exception is thrown. 

```dart
expect(() => controller.debugCheckIsDisposed(), returnsNormally);
```